src/components/Purchase.js

```diff
@@ -0,0 +1,292 @@
+      subscription,
+      creationError,
+      intl,
+    } = this.props;
+
+    const costText = intl.formatMessage({
+      id: 'app.friends_family.cost',
+      description: 'Price label',
+      defaultMessage: '$150.00 billed annually',
+    });
+
+    const productTitle = intl.formatMessage({
+      id: 'app.friends_family.product_title',
+      description: 'Individual plan product title',
+      defaultMessage: 'Individual Subscription with Rover',
+    });
+
+    const checkoutSessionErrorMessage = () => (
+      <Grid item>
+        <Alert variant="outlined" severity="error">
+          <FormattedMessage
+            id="app.friends_family.error_message_1"
+            description="First part of purchase error message"
+            defaultMessage="There was an error. Please contact"
+          />
+          {' '}
+          <Link href="mailto:support@rovercode.com">
+            support@rovercode.com
+          </Link>
+          {' '}
+          <FormattedMessage
+            id="app.friends_family.error_message_2"
+            description="Second part of purchase error message"
+            defaultMessage="to complete your purchase."
+          />
+        </Alert>
+      </Grid>
+    );
+
+    const pageContent = () => {
+      if (isFetching) {
+        return (
+          <Grid item xs={12} lg={12}>
+            <CircularProgress />
+          </Grid>
+        );
+      }
+      if (subscription.plan === '3') {
+        return (
+          <>
+            <Grid item xs={12} lg={12}>
+              <Typography>
+                <FormattedMessage
+                  id="app.friends_family.already_purchased"
+                  description="Messages explaining that the user already purchased the Individual Plan"
+                  defaultMessage="You have already purchased the Individual Plan for this account"
+                />
+              </Typography>
+            </Grid>
+            <Button
+              variant="contained"
+              color="primary"
+              component={Link}
+              to="/courses"
+            >
+              <Typography>
+                <FormattedMessage
+                  id="app.friends_family.already_purchased_action"
+                  description="Button giving an action for those who have already purchased"
+                  defaultMessage="Start your first lesson!"
+                />
+              </Typography>
+            </Button>
+          </>
+        );
+      }
+      return (
+        <>
+          <Grid item xs={12} lg={12}>
+            <Typography variant="subtitle1">
+              <FormattedMessage
+                id="app.friends_family.instructions_1"
+                description="First part of instructions for contacting support"
+                defaultMessage="We're excited to offer you early access to Rovercode!
+                Shop below, and please email"
+              />
+              {' '}
+              <Link href="mailto:support@rovercode.com">
```

Comment:
This doesn't work because it is the React Router `Link`. It needs to be the Material UI `Link`. This needed to be done on the login page as well: https://github.com/rovercode/rovercode-ui/blob/alpha/src/containers/Accounts/Login.js#L3

Reply:
Erg, I totally knew this. Fixing now.